src/wp-includes/ms-deprecated.php

```diff
@@ -646,13 +646,13 @@ function install_blog( $blog_id, $blog_title = '' ) {
 	update_option( 'home', $home );
 
 	if ( get_site_option( 'ms_files_rewriting' ) ) {
-		update_option( 'upload_path', UPLOADBLOGSDIR . "/$blog_id/files" );
+		update_option( 'upload_path', UPLOADBLOGSDIR . "/$blog_id/files", false );
 	} else {
-		update_option( 'upload_path', get_blog_option( get_network()->site_id, 'upload_path' ) );
+		update_option( 'upload_path', get_blog_option( get_network()->site_id, 'upload_path' ), false );
 	}
 
-	update_option( 'blogname', wp_unslash( $blog_title ) );
-	update_option( 'admin_email', '' );
+	update_option( 'blogname', wp_unslash( $blog_title ), true );
+	update_option( 'admin_email', '', false );
```

Comment:
Per previous feedback, these should remain autoloaded as is (and are already set up during installation).
```suggestion
	update_option( 'blogname', wp_unslash( $blog_title ) );
	update_option( 'admin_email', '' );
```